.changeset/purple-tigers-taste.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/core-services": patch
```

Comment:
`patch` should be used for fixes.. since this is a new feature, it should be `minor`..

Reply:
Fixed!